scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.option('--portal', required=True, type=str,
+              help='Portal to add the analyses to')
+@click.option('--analyses', required=True, type=str,
```

Comment:
Instead of using comma-separated lists can you use "multiple options", see this [link](http://click.pocoo.org/4/options/#multiple-options). Same comment for the rest of the interfaces.